R/series-s3-base.R

```diff
@@ -59,7 +59,8 @@ METHODS_EXCLUDE <- c(
 # TODO: support the mode argument
 #' @export
 as.vector.polars_series <- function(x, mode = "any") {
-  x$to_r_vector(ensure_vector = TRUE) |>
+  int64 <- use_option_if_missing(int64, TRUE, "double")
+  x$to_r_vector(ensure_vector = TRUE, int64 = int64) |>
```

Comment:
Is this needed?

Reply:
Yes, to ensure that the global option is correctly used in `as.vector()`, e.g. in this test:

```r
withr::with_options(
    list(polars.to_r_vector_int64 = "character"),
    {
      expect_snapshot(
        expect_identical(
          as.vector(pl$Series("a", c(1:3, NA))$cast(pl$Int64)),
          c("1", "2", "3", NA)
        )
      )
})
```